geode-core/src/test/java/org/apache/geode/internal/cache/DistributedCacheOperationTest.java

```diff
@@ -72,11 +114,112 @@ public void endOperationIsInvokedOnDistributionError() {
     assertTrue(operation.endOperationInvoked);
   }
 
+  @Test
+  public void processReplacesVersionTagNullIDs() {
+    message.process(dm);
+
+    verify(versionTag).replaceNullIDs(sender);
+  }
+
+  @Test
+  public void processSendsReplyIfAdminDM() {
+    when(dm.getDMType()).thenReturn(ClusterDistributionManager.ADMIN_ONLY_DM_TYPE);
+
+    message.process(dm);
+
+    verify(message, never()).basicProcess(dm, region);
+    verify(message).sendReply(
+        eq(sender),
+        eq(processorId),
+        eq(null),
+        eq(dm));
+  }
+
+  @Test
+  public void processSendsReplyIfLocalRegionIsNull() {
+    doReturn(null).when(message).getLocalRegionForProcessing(dm);
+
+    message.process(dm);
+
+    assertThat(message.closed).isTrue();
+    verify(message, never()).basicProcess(dm, region);
+    verify(message).sendReply(
+        eq(sender),
```

Comment:
It's not necessary to wrap the arguments here with `eq()`, so this can be simplified.